### PR TITLE
[FEATURE] Quick-fix image behavior  (4/10/21)

### PR DESCRIPTION
**Pull request by ericlowturing**
Summary:
 Quick fix of css issues for NFT images.
(Question: I am not sure why we are using css and inline styles in jsx. We should choose one to keep code readability and quality. I recommend css(scss) only)

Task:
https://zer0.io/a/network/tasks/board/e85a99cf-665f-427c-bb0b-2b4e12ba207a/2ac47463-959e-49d2-85d9-6e632dfdac92

### CODE REVIEW

File: src/components/Cards/PreviewCard/PreviewCard.module.css [hunk 1]
```diff
@@ -53,18 +53,9 @@
 	display: flex;
 	align-items: center;
 	justify-content: center;
-
 	width: 398.33px;
-	height: 220px;
```
Comment (colbr) [inline src/components/Cards/PreviewCard/PreviewCard.module.css:58]: We will need a set height for the preview card image, as the preview card is the same height for all image sizes [in the design](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=5925%3A144039). If it doesn't have a set height, the preview card will snap to a larger height after loading an image as in the video below
https://user-images.githubusercontent.com/12437916/136883894-f0f27c60-4722-432e-895f-87cb19271eb8.mov

Basically for the preview card we just need an image container with set dimensions, then the image is contained within that.

File: src/components/Cards/PreviewCard/PreviewCard.module.css [hunk 2]
```diff
@@ -53,18 +53,9 @@
...
-
-	background-color: var(--background-purple-card);
-	background: rgba(255, 255, 255, 0.05);
-	background-size: cover;
-	background-position: center;
-	background-repeat: no-repeat;
-
+	max-height: 220px;
 	border-radius: 9px 0px 0px 13px;
```
Comment (colbr) [inline src/components/Cards/PreviewCard/PreviewCard.module.css:58]: This border radius isn't applying to the image for whatever reason - the image in preview card [should have rounded edges like this](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=5925%3A144039).

File: src/containers/other/NFTView/NFTView.module.css [hunk 3]
```diff
@@ -46,11 +46,9 @@
 }
 
 .Image {
-	height: 422px;
-	min-height: 422px;
+	height: auto;
 	max-height: 422px;
 	cursor: pointer;
-	background: rgba(0, 0, 0, 0.3);
 	border-radius: inherit;
```
Comment (colbr) [inline src/containers/other/NFTView/NFTView.module.css:52]: We shouldn't have a border radius here - [relevant Figma frame](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=5959%3A118358)

File: src/components/Cards/NFTCard/NFTCard.module.css [hunk 4]
```diff
@@ -41,6 +41,7 @@
 }
 
 .NFT {
+	padding: 8px;
```
Comment (colbr) [inline src/components/Cards/NFTCard/NFTCard.module.css:44]: Won't need this padding with the cropped images -- [relevant Figma frame](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=6024%3A240680)

**Comment by colbr** [comment]
Good job @ericlowturing
